pennylane/wires.py

```diff
@@ -111,7 +114,7 @@ def __eq__(self, other):
 
     def __hash__(self):
         """Implements the hash function."""
-        return hash(repr(self.labels))
+        return hash(self.labels)
```

Comment:
This might have unexpected consequences. @mariaschuld would know better

Reply:
As far as I understand, `__hash__` is used by various methods (such as `set`) to compare instances of a class. In PennyLane we were avoiding behaviour that implicitely compares wires (instead we compare wire labels). This change should therefore be safe, and I think it is better practice anyways.